### DISCUSSION TITLE
Help, how to convert a table to text without column names?

### DISCUSSION THREAD

**Discussion by name-snrl** [nushell/nushell#13383] [General]
There no discussions about this, and I read documentation, but can't find this simple operation.

**Comment by fdncred** [comment]
It depends on what you're trying to do. There is the `values` command that may help.
```nushell
❯ [[a b c]; [1 2 3] [x y z]] | values | table -e
╭───┬───────────╮
│ 0 │ ╭───┬───╮ │
│   │ │ 0 │ 1 │ │
│   │ │ 1 │ x │ │
│   │ ╰───┴───╯ │
│ 1 │ ╭───┬───╮ │
│   │ │ 0 │ 2 │ │
│   │ │ 1 │ y │ │
│   │ ╰───┴───╯ │
│ 2 │ ╭───┬───╮ │
│   │ │ 0 │ 3 │ │
│   │ │ 1 │ z │ │
│   │ ╰───┴───╯ │
╰───┴───────────╯
```

**Reply by name-snrl** [reply]
> what you're trying to do

I'm trying to parse the output of the command, discard unnecessary information and then convert that to json:

```nu
# cmd
"one two three\none two three"
| lines
| split column ' '
| reject column1
| <something that removes header>
| to json
# output
[
  "two three",
  "two three"
]
```

**Reply by fdncred** [reply]
How about something like this? 
```nushell
❯ "one two three\none two three" | from ssv --noheaders  | get column0 | to json
[
  "one two three",
  "one two three"
]
```

**Reply by name-snrl** [reply]
> How about something like this?

but I need to drop the first column, I only need to get "two three" from each input row

**Comment by name-snrl** [comment]
okay, I did it. First, there is a better way than splitting and rejecting columns:

```nu
"one two three\none two three" | lines
| parse -r " (.*)" | values | flatten | to json
```

```nu
"one two three\none two three" | lines
| each {|v| $v | split row ' ' | drop nth 0 | str join ' '} | to json
```

but if you want to operate on columns, you can do it this way:

```nu
"one two three\nfoo bar baz" | lines
| split column ' ' | reject column1 | each {|v| $v | values | str join ' '} | to json
```

output for each command:

```
[
  "two three",
  "two three"
]
```

**Comment by NotTheDr01ds** [comment]
Sorry for the late reply.  Since you are really working with *text* here, it's probably easiest in this case to just use a regex to chop off the first word/column:

```nu
"one two three\none two three" | lines
| str replace -r '^\S* (.*)' '$1'
| to json
```

That said, the `reject column1` is probably the most Nu'ish way.
